src/layout/footer/index.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import Button from '../../components/Button'
+import classes from "./index.module.scss";
+import { FaFacebookF, FaTwitter, FaBlog } from "react-icons/fa";
+
+export default function index() {
+  return (
+    <footer className={classes["footer"]}>
```

Comment:
```suggestion
    <footer className={classes.footer}>
```
and change respectively other places